src/PrestaShopBundle/Resources/views/Admin/Product/form_combinations_bulk.html.twig

```diff
@@ -74,7 +74,9 @@
   </div>
 
   <div class="col-lg-4 col-md-3 col-sm-6">
-    <label class="form-control-label">{{ form.low_stock_threshold.vars.label }}</label>
+    <label class="form-control-label">{{ form.low_stock_threshold.vars.label }}
+      <span class="help-box" data-toggle="popover" data-content="{{ 'You can only increase or decrease low stock levels in bulk, but you cannot disable it. To disable a low stock level, do it combination by combination.'|trans({}, 'Admin.Catalog.Feature') }}" ></span>
```

Comment:
I'd change it to: "You can increase or decrease low stock levels in bulk. You cannot disable them in bulk: you have to do it on a per-combination basis."
ping @AlexEven ?